CONTRIBUTING.md

```diff
@@ -34,9 +34,12 @@ If your testing code is more extensive than unit tests (in terms of test executi
 
 Changes to NNCF Python code should conform to [Python Style Guide](./docs/styleguide/PyGuide.md)
 
-Pylint is used throughout the project to ensure code cleanliness and quality.
-A Pylint run is also done as part of the pre-commit scope - the pre-commit `pytest` scope will not be run if your code fails the Pylint checks.
-The Pylint rules and exceptions for this repository are described in the standard [.pylintrc](./.pylintrc) format - make sure your local linter uses these.
+Basic code style and static checks are enforced using a `pre-commit` Github action.
```

Comment:
Looks like it can be shortly and details describe in PyGuide.md
```
Changes to NNCF Python code should conform to [Python Style Guide](./docs/styleguide/PyGuide.md).
You can run the code style checks locally using the make pre-commit command.
For more details, see [Automating Code Formatting](./docs/styleguide/PyGuide.md#2-automating-code-formatting) and [Ruff](TBD).
```
 
I think all mentions about pylint can be deleted. Developers can use any tools while checks passes by default, don't need write about that.

Reply:
Ok, didn't notice that the section on that is already there in the pyguide